abci/cmd/abci-cli/abci-cli.go

```diff
@@ -168,7 +170,7 @@ This command opens an interactive console for running any of the other commands
 without opening a new connection each time
 `,
 	Args:      cobra.ExactArgs(0),
-	ValidArgs: []string{"echo", "info", "finalize_block", "check_tx", "commit", "query"},
+	ValidArgs: []string{"echo", "info", "finalize_block", "check_tx", "commit", "query", "prepare_proposal"},
```

Comment:
```suggestion
	ValidArgs: []string{"echo", "info", "query", "check_tx", "prepare_proposal", "finalize_block", "commit"},
```
Just to group them by functionality... not sure it is has any effect TBH